plugins/utils/mini.nix

```
@@ -1,51 +1,89 @@
 {
   lib,
-  helpers,
-  config,
-  pkgs,
   ...
 }:
-with lib;
-let
-  cfg = config.plugins.mini;
-in
-{
-  options.plugins.mini = {
-    enable = mkEnableOption "mini.nvim";
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "mini";
+  originalName = "mini.nvim";
 
-    package = lib.mkPackageOption pkgs "mini.nvim" {
-      default = [
-        "vimPlugins"
-        "mini-nvim"
-      ];
-    };
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  package = "mini-nvim";
 
-    modules = mkOption {
-      type = with types; attrsOf attrs;
+  extraOptions = {
+    modules = lib.mkOption {
+      type = with lib.types; attrsOf (attrsOf anything);
       default = { };
       description = ''
         Enable and configure the mini modules.
```

I think a paragraph break would be good? Up to you.

```suggestion
        Enable and configure the mini modules.

```